grid/recruitment/candidates/maps/breezy-hr.suma

```diff
@@ -160,3 +108,136 @@ operation MapBreezyHRError {
     rateLimit = rateLimit
   }
 }
+
+operation CreateCandidate {
+  supportedSocialProfiles = [
+    'facebook', 'linkedin', 'twitter', 'dribbble', 'instagram', 'behance', 
+    'angellist', 'flickr', 'github', 'youtube', 'google-plus', 'skype', 'globe'
+  ]
+
+  links = {}
+
+  candidate = args.candidate
+
+  tmp = (candidate.links || []).forEach(link => {
+    if (supportedSocialProfiles.includes(link.name)) {
+      links[link.name] = link.url
+    } else {
+      links['globe'] = link.url
```

Comment:
issue: you are rewriting globe with the all the social link urls that are not in the supported list. I propose to add not supported links to `custom_attributes`.